docker/install-joomla.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+TARGET=$1
+
+if [ "$TARGET" = "" ]; then
+    TARGET="joomla"
+fi
+
+docker-compose down
+docker-compose build joomla-base
+docker-compose up -d $TARGET
+
+# Wait for services to be ready
+sleep 15
+
+# Install Joomla
+docker-compose exec -e JOOMLA_DB_HOST=joomladb $TARGET joomla site:install Yoti \
+  -H joomladb \
+  --mysql-database yotijoomla \
+  --skip-create-statement \
+  --www /var/www/html \
+  --use-webroot-dir
+
+# Enable Yoti module
+if [ "$TARGET" != "joomla-base" ]; then
+  docker-compose exec -T joomladb mysql -uroot -proot yotijoomla < ./mysql-dump.sql
+fi
```

Comment:
Do you think having something like `read -n1 -r -p "Press any key to continue..." key` at the end of the `install-joomla.sh` file would be useful, in case there are any errors, so that the window persists? Since it launches in a detached window

Reply:
I know we might need to update the Travis files if we did this though..

Reply:
Disabling pseudo-tty allocation in 24a6fb5 should resolve the Windows issues